galaxy/api/v2/tests/test_collection_view.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from galaxy.main import models
+
+
+class CollectionViewTests(TestCase):
+
+    def setUp(self):
+        self.namespace = models.Namespace.objects.create(
```

Comment:
`setUp()` method is called before executing every test in test case class.

Testing database is not recreated for each test case, therefore creating namespace with non random name will cause conflict if more than one test in class or with other tests.

Reply:
In fact, the main value point of `django.test.TestCase`, which `APITesetCase` inherits, is that each test is isolated from one another by virtue of being run in a separate transaction, which is rolled back at the end of the test. So, no need for cleanup and the created objects are already removed.

Reply:
I re-checked Django tests behavior and you're right, no need to cleanup database resources between tests.